app/models/draft-registration.ts

```diff
@@ -63,6 +64,17 @@ export default class DraftRegistrationModel extends OsfModel {
 
     @hasMany('contributor')
     contributors!: DS.PromiseManyArray<ContributorModel> & ContributorModel[];
+
+    @computed('currentUserPermissions')
```

Comment:
I think we no longer need `@computed` for these properties as model attrs are all tracked now.

Reply:
@adlius could you please point me to where you got this from? AFAIK only component args `this.args.X` are auto-tracked and you need extend from `glimmer/component` to take advantage of that?